File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/main/java/gameCatan/models/Human.java
package gameCatan.models;

public class Human extends Player {

    public Human(int storageWood, int storageCoal, int storageSeed) {
        super(storageWood, storageCoal, storageSeed);
    }

    @Override
    public String toString() {
        return "Human{" +
                "storageWood=" + storageWood +
                ", storageCoal=" + storageCoal +
                ", storageSeed=" + storageSeed +
                '}';
    }
}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/src/main/kotlin/gameCatan/models/Human.kt
package gameCatan.models

class Human(storageWood: Int, storageCoal: Int, storageSeed: Int) : Player(storageWood, storageCoal, storageSeed) {
    override fun toString(): String {
        return "Human{" +
                "storageWood=" + storageWood +
                ", storageCoal=" + storageCoal +
                ", storageSeed=" + storageSeed +
                '}'
    }
}